lego/apps/events/tests/test_events_api.py

```diff
@@ -289,17 +289,29 @@ def test_event_registration_sort_order_rearranged(self):
         group1 = AbakusGroup.objects.create(name='DummyGroup1', type=GROUP_GRADE)
         group2 = AbakusGroup.objects.create(name='DummyGroup2', type=GROUP_GRADE)
         reg1, reg2, reg3 = event.registrations.all().order_by('id')[:3]
+
         group1.add_user(auth_user)
         group1.add_user(reg3.user)
         group1.add_user(reg1.user)
         group2.add_user(auth_user)
         group2.add_user(reg3.user)
 
+        shared_memberships_count = {
+            reg1.id: 1,
+            reg2.id: 0,
+            reg3.id: 2,
+        }
+
         event_response = self.client.get(_get_detail_url(5))
-        memberships = [
-            reg['sharedMemberships'] for reg in event_response.data['pools'][0]['registrations']
-        ]
-        self.assertEqual(memberships, [1, 0, 2])  # reg1 = 1, reg2 = 0, reg3 = 2
+
+        registrations = event_response.data['pools'][0]['registrations']
+
+        for reg in registrations:
+            reg_id = reg['id']
+            self.assertEqual(
+                reg['sharedMemberships'], shared_memberships_count[reg_id],
```

Comment:
Hmm, I assumed the registrations from the payload was sorted by the id in addition to the registration date, but I see that it is only sorted by the registration date. We could add ['registration_date', 'id'] to the ordering on the model to fix this, but I suppose that there isnt a problem with not having any specific ordering on identical registration_date's :)